fed/mosaic-cell/src/test/java/org/eclipse/mosaic/fed/cell/utility/RegionUtilityTest.java

```diff
@@ -19,20 +19,22 @@
 import static org.junit.Assert.assertEquals;
 
 import org.eclipse.mosaic.fed.cell.config.model.CMobileNetworkProperties;
+import org.eclipse.mosaic.fed.cell.config.model.CNetworkProperties;
 import org.eclipse.mosaic.fed.cell.data.ConfigurationData;
 import org.eclipse.mosaic.fed.cell.data.SimulationData;
 import org.eclipse.mosaic.fed.cell.junit.CellConfigurationRule;
 import org.eclipse.mosaic.fed.cell.junit.CellSimulationRule;
-import org.eclipse.mosaic.lib.geo.GeoPoint;
-import org.eclipse.mosaic.lib.geo.UtmPoint;
-import org.eclipse.mosaic.lib.geo.UtmZone;
+import org.eclipse.mosaic.lib.geo.*;
```

Comment:
It seems your IDE settings are not correct. We usually do not allow to use `*` in imports.

Reply:
Checked that and somehow the code style was not in the IDE settings anymore. Imports are corrected.